src/plugins/mf_priority.cpp

```diff
@@ -204,6 +216,74 @@ static int validate_cb (flux_plugin_t *p,
                                     "user does not belong to specified bank");
     }
 
+    mj_it = users_mj.find (userid);
+
+    if (bank == NULL) {
+        def_bank_it = users_def_bank.find (userid);
+        mj_inner_it = mj_it->second.find (def_bank_it->second);
+
+        if (flux_jobtap_job_aux_set (p,
+                                     FLUX_JOBTAP_CURRENT_JOB,
+                                     "mf_priority:bank",
+                                     &def_bank_it->second[0],
+                                     NULL) < 0)
+            flux_log_error (h, "flux_jobtap_job_aux_set");
+    } else {
+        mj_inner_it = mj_it->second.find (bank);
+
+        if (flux_jobtap_job_aux_set (p,
+                                     FLUX_JOBTAP_CURRENT_JOB,
+                                     "mf_priority:bank",
+                                     &bank[0],
```

Comment:
I can't think of a good reason to use `&bank[0]` here, I think that is the same address as just `bank`.